test/Example01.t.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {Test} from "forge-std/Test.sol";
+import {
+    CCIPLocalSimulator,
+    IRouterClient,
+    LinkToken,
+    BurnMintERC677Helper
+} from "@chainlink/local/src/ccip/CCIPLocalSimulator.sol";
+import {Client} from "@chainlink/contracts-ccip/src/v0.8/ccip/libraries/Client.sol";
+
+contract Example01Test is Test {
+    CCIPLocalSimulator public ccipLocalSimulator;
+
+    address alice;
+    address bob;
+    IRouterClient router;
+    uint64 destinationChainSelector;
+    BurnMintERC677Helper ccipBnMToken;
+    LinkToken linkToken;
```

Comment:
Nit: since this is extracted from the Simulator, for educational purposes consider whether calling the var `simulatorLinkToken` would pack more information for newbie readers.